tardis/io/logger/tests/test_logging.py

```diff
@@ -7,17 +7,34 @@
 from tardis import run_tardis
 
 
-def test_logging_simulation(atomic_data_fname, caplog):
+@pytest.fixture(scope="session")
```

Comment:
so I wonder if the fixtures can live more centrally?

Reply:
Yes, we do have fixtures in multiple `confest.py` files, which I assume are resued. (This fixture does too, I just wanted to modify it to make it even simpler)